src/content/docs/mobile-apps/dashboards-new-relic-mobile-apps.mdx

```diff
@@ -0,0 +1,116 @@
+Private: Only you can see the dashboard. Everything but the metadata is hidden.
+
+To edit permissions, please visit the New Relic website.
+
+## Manage Dashboards
+
+### Filter and refine your charts
+
+You can narrow down the information on display using the filtering function, which is a visual representation of query conditions. On any dashboard, tap the “Filter by any attribute or value” link at the top.
+
+Use the filter bar to select the values or attributes you want to see, and remove the rest of the elements from the charts.
+Open the advanced filter bar to access the boolean operators (such as =, !=, CONTAINS, EXCLUDES, etc.) and add compound and complex conditions for filtering data.
+After applying a filter, your dashboard will only show the data associated to the elements you selected. A small counter indicates the number of filters applied at a time.
+
+To return to the default view, tap on the “X” by the filter at the top to remove it.
+
+### Filter using the chart legend
+
+Tap on a legend in any chart with legends to see that series only, and remove the rest of the series from the chart. This helps you isolate the data you want to analyze.
+
+### Adjust time settings with **Time Picker**
+
+By default, each chart in the dashboard shows data for the time period specified when they were created in the query builder. However, you can use the time picker to change the time range of the data on display and set the same range for all charts. This is particularly useful while troubleshooting incidents, if you need to narrow down your data to observe what happened in a specific time period.
+
+To change the time range:
+
+Tap the clock icon at the top right of your dashboard
+Choose one of the available options from the menu by tapping radio button
+Or customize the time range with specific start and end timestamps using the custom menu.
+
+### Share your data
+
+1. In your dashboard, tap the Share icon in the top right
+2. Use your device’s share sheet to send your dashboard link
+
+### Data Scrubber
+
+The chart scrubber helps you select a data point or facet in a chart when the chart is too crowded and facets are too close to each other.
+
+Move your finger along the needle to smoothly select the adjacent facets and view their associated data points. You can also lighten a heavily populated chart by unselecting one or more of the attributes that appear in the UI.
+
+### Template variables: dynamically filter dashboards
+
+In New Relic dashboards, template variables are placeholders in the NRQL queries that can be later assigned a value, which allows for more flexibility when handling different use cases. Once you create dashboards with these variables in New Relic on the web, you can then view them in New Relic Mobile, allowing you to easily filter through your dashboard for more precise data insights.
```

Comment:
I'd add the link for [NRQL queries](https://docs.newrelic.com/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/)